test/middleware/spec.back-link.js

```diff
@@ -81,6 +81,8 @@ describe('Back Links', function () {
     });
 
     it('adds the previous step to res.locals.backLink', function () {
+        req.get.withArgs('referrer').returns('http://example.com/referrer');
+        req.baseUrl = '/referrer';
```

Comment:
`req.get.withArgs('referrer').returns('http://example.com/referrer');` does not equal `req.baseUrl = '/referrer' `